src/main/java/se/sundsvall/messaging/api/StatusAndHistoryResource.java

```diff
@@ -322,14 +321,51 @@ ResponseEntity<Statistics> getStats(
 
         return ResponseEntity.ok(stats);
     }
+
+    @Operation(
+        summary = "Get letter delivery statistics by department",
+        responses = {
+            @ApiResponse(
+                responseCode = "200",
+                description = "Successful Operation",
+                content = @Content(array = @ArraySchema(schema = @Schema(implementation = DepartmentStatistics.class)))
+            ),
+            @ApiResponse(
+                responseCode = "500",
+                description = "Internal Server Error",
+                content = @Content(schema = @Schema(implementation = Problem.class))
+            )
+        }
+    )
+    @GetMapping(
+        value = {"/statistics/departments","/statistics/departments/{department}"},
+        produces = { APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE }
+    )
+    ResponseEntity<DepartmentStatistics> getDepartmentStats(
```

Comment:
Is it correct to use @ArraySchema? The return type is not a list. Should maybe just set "useReturnTypeSchema = true" instead?